src/reanimated2/utils.ts

```diff
@@ -3,7 +3,7 @@ import { measure } from './NativeMethods';
 import type { AnimatedRef } from './hook/commonTypes';
 import type { SharedValue } from './commonTypes';
 
-export interface ComponentCoords {
+interface ComponentCoords {
```

Comment:
This should be exported as well since this is the return type of `getRelativeCoords` which is also exported.
```suggestion
export interface ComponentCoords {
```